src/dug/core/parsers/bacpac_parser.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List
+from xml.etree import ElementTree as ET
+
+from dug import utils as utils
+from ._base import DugElement, FileParser, Indexable, InputFile
+
+logger = logging.getLogger('dug')
```

Comment:
Should this be a BACPACParser-specific parser so it's easier to figure out where the log messages are coming from? Or maybe it doesn't matter?

Reply:
Hmm, that's a good point. We have a bunch of logging setups all over the place, currently all use dug logger. But we may need to do some overhaul on logging in the next round maybe as we refactor some of the NER integration aswell. For now i think changing this one would make it a special maybe (?)

Reply:
Yeah, that makes sense! It's probably worth opening an issue for this and dealing with all of these there.